redash/query_runner/uptycs.py

```diff
@@ -0,0 +1,141 @@
+import os
+import sys
+import json
+import jwt
+import datetime
+import requests
+import urllib3
+import logging
+logger = logging.getLogger(__name__)
+
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+
+enabled = True
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+class Uptycs(BaseSQLQueryRunner):
+
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string"
+                },
+                 "customerid": {
+                    "type": "string"
+                },
+                "key": {
+                    "type": "string"
+                },
+                "secret": {
+                    "type": "string",
+                },
+            },
+            "order": ['url', 'customerid', 'key', 'secret'],
+            "required": ["url", "customerid", "key","secret"],
+            "secret": ["secret", "customerid", "key"]
+        }
+
+
+    @classmethod
+    def enabled(cls):
+        return enabled
+
+    @classmethod
+    def annotate_query(cls):
+        return False
+
+
+    @classmethod
+    def type(cls):
+        return "uptycs"
+
+
+    def generateHeaders(self, key, secret):
```

Comment:
We're following [PEP-8](https://www.python.org/dev/peps/pep-0008/) as our styleguide, so this should be `generate_headers`.

Reply:
Made the changes as per your comment